storage/txcache/txCache_test.go

```diff
@@ -260,109 +268,3 @@ func Test_IsInterfaceNil(t *testing.T) {
 	thisIsNil := makeNil()
 	require.True(t, check.IfNil(thisIsNil))
 }
-
-// This seems to be the worst case in terms of eviction complexity
-// Eviction is triggered often and little eviction (only 10 senders) is done
-func Benchmark_AddWithEviction_UniformDistribution_250000x1_WithConfig_NumOldestSendersToEvict_10(b *testing.B) {
```

Comment:
these benchmarks may be kept.

Reply:
I've re-added them to `eviction_test.go`. Also added a script `benchmarks.sh` to execute them.